test/index.ts

```diff
@@ -0,0 +1,65 @@
+/*
+import { ClientConfig } from 'iota-is-client';
+import { ApiVersion } from 'iota-is-client';
+import { IdentityInternal } from 'iota-is-client';
+// import { VerifiableCredentialInternal } from 'integration-services-node/lib/models/types/verification';
+​*/
+
+import { Manager } from 'iota-is-client';
+
+async function bootstrap() {
+  try {
+    let manager = new Manager('mongodb://username:password@localhost:27017', 'PpKFhPKJY2efTsN9VkB7WNtYUhX9Utaa');
+​
+    let rootId = await manager.getRootIdentity();
```

Comment:
I think this should be obsolete + there is a lot of commented code in here

Reply:
Thanks for noticing it.
It was a test project.
Removed